pages/community.tsx

```diff
@@ -0,0 +1,397 @@
+    cards: [
+      {
+        image: "",
+        link: "https://ambassador.thirdweb.com/",
+        title: "Ambassadors",
+        description: "",
+      },
+      {
+        image: "",
+        link: "https://www.notion.so/thirdweb/University-Program-Page-e090e7cefdc14c0f8e8ad6aa47dbdb74?pvs=4",
+        title: "University Program",
+        description: "",
+      },
+      {
+        image: "",
+        link: "https://www.notion.so/thirdweb/Regional-Champions-Page-cebe9db7b2534162b0fd87e60039280c?pvs=4",
+        title: "Regional Champions",
+        description: "",
+      },
+      {
+        image: "",
+        link: "/",
+        title: "Showcase",
+        description: "",
+      },
+    ],
+  },
+];
+
+const galleryImages = [
+  {
+    src: "/assets/community/gallery/1.png",
+  },
+  {
+    src: "/assets/community/gallery/2.png",
+  },
+  {
+    src: "/assets/community/gallery/3.png",
+  },
+  {
+    src: "/assets/community/gallery/4.png",
+  },
+  {
+    src: "/assets/community/gallery/5.png",
+  },
+  {
+    src: "/assets/community/gallery/6.png",
+  },
+  {
+    src: "/assets/community/gallery/7.png",
+  },
+  {
+    src: "/assets/community/gallery/8.png",
+  },
+  {
+    src: "/assets/community/gallery/9.png",
+  },
+];
+
+const Community: ThirdwebNextPage = () => {
+  const trackEvent = useTrack();
+
+  return (
+    <DarkMode>
+      <NextSeo {...SEO} />
+      <Flex
+        sx={{
+          // overwrite the theme colors because the home page is *always* in "dark mode"
+          "--chakra-colors-heading": "#F2F2F7",
+          "--chakra-colors-paragraph": "#AEAEB2",
+          "--chakra-colors-borderColor": "rgba(255,255,255,0.1)",
+          "--product-accent-color": "rgba(24, 67, 78, 0.8)",
+        }}
+        justify="center"
+        flexDir="column"
+        as="main"
+        bg="#000"
+      >
+        <HomepageTopNav />
+        {/* pull it up by as much as the topnav is tall */}
+        <Box mt="-80px" overflowX="hidden">
+          <Center
+            py={{ base: 12, md: 24 }}
+            px={{ base: 4, md: 8 }}
+            flexDir="column"
+          >
+            <Heading mt={8} textAlign="center" fontSize="64px">
+              A decentralized internet begins
+              <br /> with{" "}
+              <Text
+                as="span"
+                fontSize="48px"
+                fontWeight="bold"
+                background="linear-gradient(243.9deg, #BFA3DA 21.81%, #84309C 48.81%, #C735B0 86.61%), #ECECEC"
+                backgroundClip="text"
+                color="transparent"
+              >
+                decentralized talent.
+              </Text>
+            </Heading>
+
+            <LinkButton
+              href="https://discord.gg/thirdweb"
+              size="lg"
+              onClick={() =>
+                trackEvent({
+                  category: TRACKING_CATEGORY,
+                  action: "click",
+                  title: "Join the community",
```

Comment:
```suggestion
                  label: "join-the-community",
```

always have a label and make it slug-like